app/src/main/java/org/simple/clinic/editpatient/EditPatientModel.kt

```diff
@@ -1,3 +1,49 @@
 package org.simple.clinic.editpatient
 
-object EditPatientModel
+import org.simple.clinic.patient.Gender
+import org.simple.clinic.patient.Patient
+import org.simple.clinic.patient.PatientAddress
+import org.simple.clinic.patient.PatientPhoneNumber
+import org.threeten.bp.format.DateTimeFormatter
+
+data class EditPatientModel(
+    val savedEntry: OngoingEditPatientEntry,
```

Comment:
Discussed while pairing: we should rename `OngoingPatientEntry` to `EditableFields`.